src/plugins/graph/models/graph-model.ts

```diff
@@ -461,25 +461,20 @@ export const GraphModel = TileContentModel
       }
 
       // Display a plotted function when this is linked to a SharedVariableModel
-      addDisposer(self, reaction(
-        () => {
-          const smm = getSharedModelManager(self);
-          let sharedVariableModels;
-          if (smm?.isReady) {
-            sharedVariableModels = smm.getTileSharedModelsByType(self, SharedVariables);
-          }
-          return sharedVariableModels;
-        },
-        (sharedVariableModels) => {
-          if (sharedVariableModels && sharedVariableModels.length > 0) {
-            const plottedFunctionAdornment = PlottedFunctionAdornmentModel.create();
-            plottedFunctionAdornment.addPlottedFunction(x => x**2);
-            self.showAdornment(plottedFunctionAdornment);
-          } else {
-            self.hideAdornment(kPlottedFunctionType);
-          }
+      addDisposer(self, autorun(() => {
```

Comment:
This may be future work, but just noting that we discussed possibly moving this into the graph's `updateAfterSharedModelChanges`